frontend/express/public/core/user-activity/javascripts/countly.views.js

```diff
@@ -24,6 +24,8 @@
                 set: function(value) {
                     this.$store.dispatch('countlyUserActivity/onSetUserActivityFilters', value);
                     this.$store.dispatch('countlyUserActivity/fetchAll', true);
+                    //TODO: when cly-qb-bar component supports filters without having to build them, set the url parameters to filter value
```

Comment:
What does this mean exactly? :slightly_smiling_face:

Reply:
The problem which I mentioned in reference to qb filter parameters incompatibility between the server and the component. Now that the config property is in the component, I can add the filter value in the URL, which should load  in the qb component as well as succesfully fetch the data in the server.